carbon-design-system/carbon-addons-iot-react
# fix(FlyoutMenu): Flyout menu position

Closes #3362 #3633 

**Summary**

FlyoutMenu does not move when parent is scrolling. 

**Change List (commits, features, bugs, etc)**

- added css attribute `--top` and `--left` so that FlyoutMenu can move with the parent.
- temporarily updated the single story and absolute story to test. I will change it back later.
- fixed style PropTypes

**Acceptance Test (how to verify the PR)**

- go to single select [story](https://deploy-preview-3627--carbon-addons-iot-react.netlify.app/?path=/story/2-watson-iot-experimental-%E2%98%A2%EF%B8%8F-datetimepickerv2--single-select)
- click on the Calendar button to open flyout menu
- scroll up and down in the red container
- scroll left and right in the red container
- notice the flyout menu is moving with the datetime input

https://user-images.githubusercontent.com/24279794/199554269-aee7c213-1872-44ea-8fa9-a335c309857d.mov



**Regression Test (how to make sure this PR doesn't break old functionality)**

- tests here



**Things to look for during review**
